### DISCUSSION TITLE
How to filter array data type field.

### DISCUSSION THREAD

**Discussion by EmreKiyat** [supabase/supabase#30767] [Questions] [answered]
Hello,
i can not build equivalent of my sql: (pattern_arr field is an array type of text)
_SELECT
  *
FROM myTable
WHERE
pattern_arr[1] in ('N','C')_

try to something like that but it did't worked(programming language is swift);

           _response = try await supaBase
               .from(Table.myTable)
               .select()
               .in("pattern_arr[1]", values: ["V","C"] )_

best regards.

**Comment by GaryAustin1** [comment, marked as answer]
I don't know about Swift but in JS this works...
`from('table').select('*').in('testa->>0', ['cat','dog'])`

**Reply by EmreKiyat** [reply]
thanks a lot. this solution also worked for swift :)
